README.md

```diff
@@ -1 +1,58 @@
-# ToDo-List-2
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# To-Do-List
+ To-Do list webPage
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://rebel216.github.io/To-Do-List/dist)
+
+
+
+## Getting Started
+
+**WebPage to Create list of Todo Tasks to manage your day.No login and registration required .Just use the Webpage.**
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@rebel216](https://github.com/rebel216)
+- Twitter: [@rebel216](https://twitter.com/rebel216)
+- LinkedIn: [rebel216](https://linkedin.com/in/rebel216)
+
+👤 **Author2**
+
+- GitHub: [@kiboma2021](https://github.com/kiboma2021)
+- Twitter: [@BenOmayio](https://twitter.com/omayiobenj)
+- LinkedIn: [@BenOmayio](https://www.linkedin.com/in/ben-omayio-74622469/)
+
```

Comment:
[ OPTIONAL ] I think this milestone is individual so you don't need to add information for the another author so kindly remove it from your README ✅